### ISSUE TITLE
It's unclear whether to input the full title or just the name for councillors

### ISSUE THREAD

**Issue by equivalentideas** [openaustralia/planningalerts#1224]
Extracted from #1199 

> and not including titles like "Dr", "Clr", "Ms" etc.

We just tested this with someone and they assumed that in the "Full Name" field, we did want the full title of the councillor, e.g.:

* Lord Mayor Alderman Sue Hickey
* Deputy Lord Mayor Alderman Ron Christie
* Alderman Marti Zucco

We only want the name, e.g. Sue Hickey.

They were unsure about what to do, and then decided to add the full title because "other wise people wont know they're the lord mayor".

![screen shot 2017-08-16 at 5 08 01 pm](https://user-images.githubusercontent.com/1239550/29351302-8a9b7080-82a5-11e7-9d17-092f89017eb4.png)

Admin @henare then had to manually remove titles from full names before processing the contribution.

What are the different ways we could answer this question for people? I think it should be clear up front to the person, rather than some kind of validation, because the person was actively unsure and asking the question as they were collecting the name.

**Comment by hisayohorie** [comment]
Addressed in #1234

**Comment by equivalentideas** [comment]
We've now got some instructions that people can bring up on the page, which includes a description of how the name should be formatted.

![screen shot 2017-08-24 at 4 38 41 pm](https://user-images.githubusercontent.com/1239550/29652852-bc71e3dc-88ea-11e7-9b30-17763ab8cc40.png)

I tested with a potential contributor that they did open the instructions before starting to collect information. They asked me what order the names should be in, e.g. "Hicks, Sue" or "Sue Hicks" so I added instructions for that in https://github.com/openaustralia/planningalerts/commit/1fc437582ab2000e5d7bc31f5e02b3d0b532fa02

We should reopen this if we continue to receive data in the format we don't want.
